### ISSUE TITLE
Typing once again 

### ISSUE THREAD

**Issue by freeseacher** [HeavyHorst/remco#49]
I am doing templating for service that want to get int as input
in consul key i have an int but after {{ config | toYaml }} value became string. 
service don't accept string as value :(

**Comment by HeavyHorst** [comment]
I added parseInt and parseFloat filters (https://github.com/HeavyHorst/remco/blob/master/docs/content/template/template-filters.md).

```
{{ "12000" | parseInt }}
{{ "12000.45" | parseFloat }}
```

**Comment by freeseacher** [comment]
Yep tnx. That will help me a bit, but real problem is
for
`/test` 
```
- a: 10
- b: "10"
```
and 
`/test2`
```
{ "a": 10, "b": "10"}
```
with
```
{{ getv("/test") | parseYAML | toYAML }}
{{ getv("/test2") | parseJSON | toJSON }}
```
i got 
```
- a: "10"
- b: "10"

{"a":"10","b":"10"}
```

**Comment by freeseacher** [comment]
Oh. i see the motivation here https://github.com/HeavyHorst/remco/commit/69ad5f81c29b3710cdf5f08df7759f1345a9ffb5 

and switching back to yaml.Unmarshal  with 
```
+++ b/pkg/template/template_filters.go
@@ -19,6 +19,8 @@ import (
        "strconv"
        "strings"
 
+       "gopkg.in/yaml.v2"
+
        "github.com/HeavyHorst/memkv"
        "github.com/HeavyHorst/pongo2"
        "github.com/dop251/goja"
@@ -191,7 +193,7 @@ func filterUnmarshalYAML(in *pongo2.Value, param *pongo2.Value) (*pongo2.Value,
        }
 
        var ret interface{}
-       if err := yaml_mapstr.Unmarshal([]byte(in.String()), &ret); err != nil {
+       if err := yaml.Unmarshal([]byte(in.String()), &ret); err != nil {
                return nil, &pongo2.Error{
                        Sender:    "filterUnmarshalYAML",
                        OrigError: err,
```
fixes my problem with yaml. but creates another with
```
[0000] ERROR ~/go/src/github.com/HeavyHorst/remco/bin/remco[428970]: createStageFileAndSync failed: create stage file failed: template execution failed: [Error (where: filterToJSON) | Line 1 Col 32 near 'toJSON'] json: unsupported type: map[interface {}]interface {} resource=aaa
```

**Comment by freeseacher** [comment]
Should we just make typesafe version of fromYAML or do some kind of fix in toJSON ?

**Comment by HeavyHorst** [comment]
Could you test with latest master again?

**Comment by freeseacher** [comment]
yep. works as a charm. tnx!

**Comment by freeseacher** [comment]
Any chance to release new version with that changes :) ? looks like it's time

**Comment by HeavyHorst** [comment]
I just released v0.12.0. 
Sorry for the delay.
